### DISCUSSION TITLE
LaTeX not rendering in jupyter notebook markdown cells

### DISCUSSION THREAD

**Discussion by KNOXZhao** [microsoft/vscode-jupyter#15012] [Questions and Answers]
I can render math mode in markdown files (ends with .md), but the markdown blocks in jupyter notebook are not working properly. I have tried to uninstall and reinstall all the related extensions but I still have the problem now. I asked some friends and they were unable to repro my problem.

Jupyter Extension v2024.1.2024011201 Pre-Release
<img width="823" alt="1" src="https://github.com/microsoft/vscode-jupyter/assets/90520126/b56d915c-2eb9-4f16-be30-37b71c5d7b3d">
<img width="874" alt="2" src="https://github.com/microsoft/vscode-jupyter/assets/90520126/858f058b-59bd-4912-b33b-122716ae9a2b">

Updated 2024-01-14:
Setting json of jupyter notebook:
{
    "editor.inlineSuggest.enabled": true,
    "workbench.colorTheme": "One Dark Pro Darker",
    "workbench.iconTheme": "material-icon-theme",
    "github.copilot.enable": {
        "*": true,
        "plaintext": true,
        "markdown": true,
        "scminput": false
    },
    "vscode_custom_css.imports": [
        "file://c:\\Users\\15712\\.vscode-insiders\\extensions\\brandonkirbyson.vscode-animations-1.0.7/dist/updateHandler.js"
    ],
    "explorer.confirmDelete": false,
    "liveServer.settings.donotShowInfoMsg": true,
    "git.ignoreLegacyWarning": true,
    "git.ignoreMissingGitWarning": true,
    "window.zoomLevel": 1,
    "jupyter.askForKernelRestart": false,
    "notebook.formatOnSave.enabled": true,
    "files.autoSave": "afterDelay",
    "editor.defaultFormatter": "ms-python.autopep8",
    "editor.formatOnSave": true,
    "editor.tabSize": 4,
    "editor.cursorSmoothCaretAnimation": "on"
}

**Comment by DonJayamanne** [comment]
@amunger /cc

**Comment by amunger** [comment]
That's pretty strange, and I'm not able to repro either. @mjbvz - Do you know what might cause this?

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?
